.gitignore

```diff
@@ -1,7 +1,15 @@
+######## STUFF I ADDED BETWEEN POUND SIGNS
```

Comment:
You can probably remove these comments since `git blame` can always be used to see who added what..